docs/datamodel/indexes.rst

```diff
@@ -37,9 +37,13 @@ notation shorthand <ref_dot_notation>`: ``.name``.
       index on (.name);
     }
 
-By indexing on ``User.name``, queries that filter by the ``name`` property will
-be faster, as the database can look up a name in the index instead of scanning
-through all Users sequentially.
+By indexing on ``User.name``, queries that filter, order, or group by the 
```

Comment:
Oh, I just noticed that we say "order" here, but "sort" in the other two places. I kinda like "order" since that's the related keyword, even though "sort" is probably the better verb. At any rate, just being consistent is all I really wanted to call out here, so whichever direction makes the most sense to y'all is good with me.